main.cpp

```diff
@@ -472,9 +338,129 @@ After you have MOVED your 10 UDTs, send me a DM with your pull request link.
 I will review the pseudo-code that you have written.
 */
 
+/*
+Thing 1) Rock Band
+5 properties:
+    1) number of players
+    2) number of rehersal time per week
+    3) number of live show time per week
+    4) amount of money made per live show
+    5) amount of money spent for rehersal in studio
+3 things it can do:
+    1) schedule the tour plan
+    2) do live show
+    3) make money
+
+Thing 2) Do YouTube
+5 properties:
+    1) hour amount on script
+    2) hour amount on shooting
+    3) hour amount on editing
+    4) view amount per video
+    5) money amount made from per 1000 view
+3 things it can do:
+    1) make video
+    2) increase follower
+    3) make money
 
+Thing 3) Song Writing
+5 properties:
+    1) number of words in lyrics
+    2) number of instruments used while producing
+    3) number of BPM
+    4) second amount of the song
+    5) number of people who listened
+3 things it can do:
+    1) reach to the audience
+    2) increase follower
+    3) nail record label's contract
 
+Thing 4) Product Management
```

Comment:
This is not an ideal object.  
it is too abstract. 
Your properties and actions are describing a **Product Manager.**  
- [ ] change the Thing